pkg/deploy/defaults.go

```diff
@@ -13,41 +13,47 @@
 package deploy
 
 const (
-	DefaultCheServerImageRepo        = "eclipse/che-server"
-	DefaultCodeReadyServerImageRepo  = "registry.redhat.io/codeready-workspaces/server-rhel8"
-	DefaultCheServerImageTag         = "7.0.0-RC-2.0"
-	DefaultCodeReadyServerImageTag   = "1.2"
-	DefaultCheFlavor                 = "che"
-	DefaultChePostgresUser           = "pgche"
-	DefaultChePostgresHostName       = "postgres"
-	DefaultChePostgresPort           = "5432"
-	DefaultChePostgresDb             = "dbche"
-	DefaultPvcStrategy               = "common"
-	DefaultPvcClaimSize              = "1Gi"
-	DefaultIngressStrategy           = "multi-host"
-	DefaultIngressClass              = "nginx"
-	DefaultPluginRegistryUrl         = "https://che-plugin-registry.openshift.io"
-	DefaultUpstreamPluginRegistryUrl = "https://che-plugin-registry.openshift.io/v3"
-	DefaultDevfileRegistryUrl        = "https://che-devfile-registry.openshift.io"
-	DefaultKeycloakAdminUserName     = "admin"
-	DefaultCheLogLevel               = "INFO"
-	DefaultCheDebug                  = "false"
-	DefaultPvcJobsImage              = "registry.redhat.io/ubi8-minimal:8.0-127"
-	DefaultPvcJobsUpstreamImage      = "registry.access.redhat.com/ubi8-minimal:8.0-127"
-	DefaultPostgresImage             = "registry.redhat.io/rhscl/postgresql-96-rhel7:1-40"
-	DefaultPostgresUpstreamImage     = "centos/postgresql-96-centos7:9.6"
-	DefaultKeycloakImage             = "registry.redhat.io/redhat-sso-7/sso73-openshift:1.0-11"
-	DefaultKeycloakUpstreamImage     = "eclipse/che-keycloak:7.0.0-RC-2.0"
-	DefaultJavaOpts                  = "-XX:MaxRAMFraction=2 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 " +
+	DefaultCheServerImageRepo           = "eclipse/che-server"
+	DefaultCodeReadyServerImageRepo     = "registry.redhat.io/codeready-workspaces/server-rhel8"
+	DefaultCheServerImageTag            = "7.0.0-RC-2.0"
+	DefaultCodeReadyServerImageTag      = "1.2"
+	DefaultCheFlavor                    = "che"
+	DefaultChePostgresUser              = "pgche"
+	DefaultChePostgresHostName          = "postgres"
+	DefaultChePostgresPort              = "5432"
+	DefaultChePostgresDb                = "dbche"
+	DefaultPvcStrategy                  = "common"
+	DefaultPvcClaimSize                 = "1Gi"
+	DefaultIngressStrategy              = "multi-host"
+	DefaultIngressClass                 = "nginx"
+	DefaultPluginRegistryImage          = "quay.io/eclipse/che-devfile-registry:7.0.0-RC-2.0"
```

Comment:
same again. rc-3.0?

Reply:
yeah, @benoitf noticed that and provided a PR to fix it, which I already merged.